lib/membrane_http_adaptive_stream/manifest.ex

```diff
@@ -64,8 +73,9 @@ defmodule Membrane.HTTPAdaptiveStream.Manifest do
   @doc """
   Restores all the stale segments in all tracks.
 
-  All the tracks must be configured to be 'persist'ed.
+  All the tracks must be configured to be 'persist'ed, otherwise this function will raise
```

Comment:
It looks quite strange tbh. 
```suggestion
  All the tracks must be configured to be persisted beforehand, otherwise this function will raise.
```